e2e/panels-suite/panelEdit_base.spec.ts

```diff
@@ -46,11 +46,16 @@ e2e.scenario({
         //  Can change to Alerts tab (graph panel is the default vis so the alerts tab should be rendered)
         e2e.components.Tab.title('Alert').should('be.visible').click();
```

Comment:
I'm not sure if I understand why this one is not failing – if Unified Alerting is enabled should this not fail? We're either loading `Tab` for legacy alerting or `AlertTab` for unified. 🤔

Reply:
It turns out that `PanelAlertTab` is still a `Tab` 👍